### PR TITLE
refact: split up manager drop box content into component files

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: src/components/manager/dropBox/FileUploadModal.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+const FileUploadModal = ({ initialUploadFolder, initialUploadFiles, onCancel, open }: FileUploadModalProps) => {
+  const dispatch = useAppDispatch();
+  const [form] = Form.useForm();
+
+  const isPutting = useDropBox().isPuttingFlow;
+
+  useEffect(() => {
+    if (open) {
+      // If we just re-opened the model, reset the fields
```
Comment (v-rocheleau) [inline src/components/manager/dropBox/FileUploadModal.tsx:31]: typo: model -> modal

File: src/components/manager/dropBox/FileUploadForm.js
Comment (v-rocheleau) [inline src/components/manager/dropBox/FileUploadForm.js:1]: Can we make this a `.tsx` file?

File: src/components/manager/dropBox/FileContentsModal.js
Comment (v-rocheleau) [inline src/components/manager/dropBox/FileContentsModal.js:1]: Can we make this a `.tsx` file?

**Review by v-rocheleau — CHANGES_REQUESTED** [review]

File: src/components/manager/dropBox/FileUploadForm.js
Comment (davidlougheed) [reply src/components/manager/dropBox/FileUploadForm.js:1]: typing for the file upload events is a bit odd but i'll give it a shot

File: src/components/manager/dropBox/FileContentsModal.js
Comment (davidlougheed) [reply src/components/manager/dropBox/FileContentsModal.js:1]: will do!

File: src/components/manager/dropBox/FileUploadForm.js
Comment (v-rocheleau) [reply src/components/manager/dropBox/FileUploadForm.js:1]: Im fine with js if typing it is too much trouble

Comment (davidlougheed) [reply src/components/manager/dropBox/FileUploadForm.js:1]: done!

File: src/components/manager/dropBox/FileContentsModal.tsx [hunk 2]
```diff
@@ -15,11 +15,20 @@ const generateURIsByRelPath = (entry, acc) => {
   return acc;
 };
 
-const FileContentsModal = ({ selectedFilePath, open, onCancel }) => {
+type FileContentsModalType = {
```
Comment (v-rocheleau) [inline src/components/manager/dropBox/FileContentsModal.tsx:18]: Would rename this to `FileContentsModalProps` for consistency

**Review by v-rocheleau — CHANGES_REQUESTED** [review]
Minor request, otherwise lgtm!

**Review by v-rocheleau — APPROVED** [review]
LGTM!

**Pull request merged**